setup.py

```diff
@@ -8,6 +8,10 @@
 with codecs.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
```

Comment:
On pip this line will fail during install - it looks for requires.txt, so while Ilke this approach I usually don't do it.